test/test_openreg.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
```

Comment:
Hi, even if we extract the `openreg` logic from `run_test.py` into a separate file, the problem persists.

https://github.com/KarhouTam/pytorch/blob/e67e3d95f3ab863c8b44a6b16ce9f25d40b0d517/tools/testing/discover_tests.py#L72

`discover_tests` can automatically find `test_openreg` and add it to the default TESTS collection.

Therefore, I think we can leave it as it is.